modules/gapi/include/opencv2/gapi/gtyped.hpp

```diff
@@ -26,6 +26,7 @@ namespace detail
     template<> struct ProtoToParam<cv::GMat>    { using type = cv::Mat; };
     template<> struct ProtoToParam<cv::GScalar> { using type = cv::Scalar; };
     template<typename U> struct ProtoToParam<cv::GArray<U> >  { using type = std::vector<U>; };
+    template<> struct ProtoToParam<cv::GArray<cv::GMat>>      { using type = std::vector<cv::Mat>; };
```

Comment:
maybe it is woth aligning by ProtoToParam<> too.

Reply:
I decided not to align like that because the line will be too long otherwise

Reply:
ok